tests/integration/s2n_dynamic_record_size_test.py

```diff
@@ -118,6 +118,10 @@ def try_dynamic_record(endpoint, port, cipher, ssl_version, threshold, server_ce
     file_input = open(test_file)
     s2nc = subprocess.Popen(s2nc_cmd, stdin=file_input, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
 
+    # Wait file send complete
+    s2nc.wait()
+    cleanup_processes(s_server)
+
```

Comment:
These tests were consistently failing. In case it was some kind of race condition between reading/writing the output, I moved the output check to after the process completes. That seems to have worked.